vagrant/Install-on-Centos-7.sh

```diff
@@ -133,7 +135,7 @@ sudo sed -i 's:#.*::' /etc/httpd/conf.d/nominatim.conf #DOCS:
 # following SELinux labeling should be done for Nominatim:
 
     sudo semanage fcontext -a -t httpd_sys_content_t "$USERHOME/Nominatim/(website|lib|settings)(/.*)?"
-    sudo semanage fcontext -a -t lib_t "$USERHOME/Nominatim/module/nominatim.so"
+    sudo semanage fcontext -a -t lib_t "$USERHOME/build/module/nominatim.so"
```

Comment:
line 137 and 139 need to be moved to build as well.

Reply:
also needed two more lines so `curl` returned results after a build.